project/backend/src/test/java/com/google/sps/RestaurantScorerTest.java

```diff
@@ -79,23 +79,24 @@ public void restaurantMissingField_doNotConsider() throws JSONException {
       /* priceNum= */ MISSING_FIELD, SOME_TYPES);
 
     double actualScore = RestaurantScorer.score(restaurantMissingField, PREFS_ALL_FIELDS, STATS);
-    double expectedScore = (EXPECTED_RATING_SCORE + WEIGHT_3 + WEIGHT_4) / (1 + WEIGHT_3 + WEIGHT_4);
+    double expectedScore = (EXPECTED_RATING_SCORE + WEIGHT_3 + WEIGHT_4 + 0.4) / (1 + WEIGHT_3 + WEIGHT_4);
+    expectedScore = expectedScore > 1 ? 1.0 : expectedScore;
     assertThat(actualScore).isEqualTo(expectedScore);
   }
 
-  @Test
-  public void restaurantOutOfBounds_subtractFromScore() throws JSONException {
-    JSONObject preferences = new JSONObject()
-      .put("currLocation", new JSONObject().put("lat", COORDS.get("lat") + 0.25).put("lng", COORDS.get("lng") + 0.25))
-      .put("radius", new JSONObject().put("pref", 10).put("weight", WEIGHT_4));
-      // Distance calculated using Haversine formula calculator: http://www.movable-type.co.uk/scripts/latlong.html
-      double percentDistDiff = 1.423969;
+  // @Test
+  // public void restaurantOutOfBounds_subtractFromScore() throws JSONException {
+  //   JSONObject preferences = new JSONObject()
+  //     .put("currLocation", new JSONObject().put("lat", COORDS.get("lat") + 0.25).put("lng", COORDS.get("lng") + 0.25))
+  //     .put("radius", new JSONObject().put("pref", 10).put("weight", WEIGHT_4));
+  //     // Distance calculated using Haversine formula calculator: http://www.movable-type.co.uk/scripts/latlong.html
+  //     double percentDistDiff = 1.423969;
 
-      double actualScore = RestaurantScorer.score(RESTAURANT_ALL_FIELDS, preferences, STATS);
-      double expectedScore = (EXPECTED_RATING_SCORE - percentDistDiff * WEIGHT_4) / (1 + WEIGHT_4);
-      // Account for margin of error in rounding from online calculator.
-      assertThat(actualScore).isWithin(0.001).of(expectedScore);
-  }
+  //     double actualScore = RestaurantScorer.score(RESTAURANT_ALL_FIELDS, preferences, STATS);
+  //     double expectedScore = (EXPECTED_RATING_SCORE - percentDistDiff * WEIGHT_4) / (1 + WEIGHT_4);
+  //     // Account for margin of error in rounding from online calculator.
+  //     assertThat(actualScore).isWithin(0.001).of(expectedScore);
+  // }
```

Comment:
uncomment?

Reply:
Oops yeah this test is no longer relevant since we changed to nearby search so I meant to remove it